test/v2/base/CommonGenerators.scala

```diff
@@ -74,4 +75,15 @@ trait CommonGenerators {
     Gen.oneOf(MovementType.values)
   }
 
+  implicit lazy val genMovementResponse: Arbitrary[MovementResponse] = Arbitrary {
```

Comment:
Name should be start with arbitrary. as this is arbitrary not gen. Like arbitraryMovementResponse